mbuild/examples/test_examples.py

```diff
@@ -15,6 +16,8 @@
 EXAMPLE_NOTEBOOKS = [f for f in glob.glob('mbuild/examples/*/*.ipynb')]
 
 
+@pytest.mark.skipif(sys.platform in ['win32', 'cygwin'], 
```

Comment:
Should we make this specific to the windows platform used by appveyor? It's not like these tests fail generally, just some odd appveyor issue.